hal/armv7a/imx6ull/_init.S

```diff
@@ -88,6 +88,17 @@ plugin:
 	/* note: "ldr reg, =constant" pseudo instruction can't be used before
 	 * the serial download check below - the literal pool will not be included in the plugin */
 
+	/* init watchdog */
+	mov32 r4, 0x20bc000
+#if defined(WATCHDOG)
+	/* 128 sec timeout, the clock is already enabled */
+	movw r5, #0xff34
```

Comment:
can we change it to `#0xff36` to enable WDBG bit by default (_Watchdog DEBUG Enable. Determines the operation of the WDOG during DEBUG mode_ - 1 means Suspend WDG during DEBUG mode).

Makes debugging using JLink much easier :)

Reply:
Does it have any side effects? If so, we might want to do this only if `NDEBUG` is not defined

Reply:
no side effects (affects only DEBUG mode). Still I would like to have it always enabled I might want to debug userspace app regardless of kernel build version.

Alternative approach is to disable wdog in gdb (only allowed in DEBUG mode):
```gdb
set *imx6ull_common.wdog &= ~0x4
```